scenarios/kubernetes_kops_aws.py

```diff
@@ -168,6 +187,17 @@ def main(args):
         '--kops-ssh-key=%s' % aws_ssh,
     ]
 
+    if args.build is not None:
```

Comment:
I think what you're trying to do is this:
https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py#L488
https://docs.python.org/2/library/argparse.html#nargs